# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xz/0002-opensde-use-automake-1.11.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

OpenSDE specific workaround because we are using automake 1.11.x on master
at this moment, which doesn't support the 'serial-tests' option.

--- ./configure.ac.orig	2013-09-07 17:34:33.046072459 +0200
+++ ./configure.ac	2013-09-07 17:35:24.983254426 +0200
@@ -418,7 +418,7 @@
 
 # Automake 1.10 should still be enough but you need to omit serial-tests.
 dnl AM_INIT_AUTOMAKE([1.10 foreign tar-v7 filename-length-max=99])
-AM_INIT_AUTOMAKE([1.12 foreign tar-v7 filename-length-max=99 serial-tests])
+AM_INIT_AUTOMAKE([1.11 foreign tar-v7 filename-length-max=99])
 AC_PROG_LN_S
 
 AC_PROG_CC_C99
